<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_widget</class>
 <widget class="QWidget" name="Main_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>912</width>
     <height>347</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="horizontalSpacing">
     <number>7</number>
    </property>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButton_draw_dist">
      <property name="text">
       <string>Нарисовать распределение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_a">
            <property name="text">
             <string>Параметр a:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_b">
            <property name="text">
             <string>Параметр b:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_enter_a"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_enter_b"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_execute">
        <property name="text">
         <string>Выполнить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_friq_elem">
      <property name="text">
       <string>Частота элементов в интервалы:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QTextEdit" name="textEdit_prob">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_num_elements">
        <property name="text">
         <string>Кол-во элементов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_num_elements"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_list_of_gen_number">
      <property name="text">
       <string>Список сгенерированных элементов</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_prob">
      <property name="text">
       <string>Теор. вероятность попадения</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_draw_elements">
      <property name="text">
       <string>Нарисовать выборку</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QTextEdit" name="textEdit_friq_elements">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_draw_histogram">
      <property name="text">
       <string>Гистограмма</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_mean">
        <property name="text">
         <string>Значение среднего:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_std">
        <property name="text">
         <string>Значение std:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="textEdit_show_list_of_generated_nums">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_mean"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_std"/>
      </item>
     </layout>
    </item>
    <item row="3" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_hi_sqr">
        <property name="text">
         <string>Значение хи-квадрат</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_hi_sqr_value">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
